libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -209,14 +249,29 @@ class AppPrompt {
     document.cookie = `dismissedAppPrompts=${JSON.stringify([...dismissedPrompts])};path=/`;
   };
 
-  close = ({ saveDismissal = true } = {}) => {
+  close = ({ saveDismissal = true, dismissalActions = true } = {}) => {
     const appPromptElem = document.querySelector(CONFIG.selectors.prompt);
     appPromptElem?.remove();
     clearTimeout(this.redirectFn);
     if (saveDismissal) this.setDismissedPrompt();
     document.removeEventListener('keydown', this.handleKeyDown);
     this.anchor?.focus();
     this.anchor?.removeEventListener('click', this.close);
+
+    const appSwitcher = document.querySelector('#unav-app-switcher');
```

Comment:
We likely don't want to maintain references to 3rd party code that might change at any time. The Unav already offers a way to fetch the element via `window.UniversalNav.getComponent?.('app-switcher')` which was already used for the PEP logic in another context.

Reply:
Yeah I missed that. Fixed now.